doc/reference.md

```diff
@@ -183,6 +184,31 @@ if (await ReaderSdk.isAuthorized) {
 ```
 
 
+---
+
+### isAuthorizationInProgress
+
+Used to determine if Reader SDK authorization is in progress.
```

Comment:
I assume this description align with the one in Reader SDK doc

Reply:
yes, copied from ios doc.